plutus-scb/app/Main.hs

```diff
@@ -370,9 +377,12 @@ runCliCommand _ _ _ (ReportContractHistory uuid) = do
 runCliCommand _ _ _ (ProcessContractInbox uuid) = do
     logInfo ProcessInboxMsg
     Core.processContractInbox @ContractExe (ContractInstanceId uuid)
-runCliCommand _ _ _ ProcessAllContractOutboxes = do
-    logInfo ProcessAllOutboxesMsg
-    Core.processAllContractOutboxes @ContractExe
+runCliCommand _ Config{requestProcessingConfig} _ ProcessAllContractOutboxes = do
+    let RequestProcessingConfig{requestProcessingInterval} = requestProcessingConfig
+    logInfo $ ProcessAllOutboxesMsg requestProcessingInterval
+    forever $ do
+        _ <- liftIO . threadDelay . fromIntegral $ toMicroseconds requestProcessingInterval
+        handleError @SCBError (Core.processAllContractOutboxes @ContractExe Trace.defaultMaxIterations) (logError . ContractExeSCBError)
```

Comment:
It seems wrong that this depends on the trace module, perhaps the default should move somewhere else?

Reply:
I moved it to `Language.Plutus.Contract.Trace.RequestHandler`